<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pauseWindow</class>
 <widget class="QDialog" name="pauseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_resume">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;MS Sans Serif&quot;;
</string>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 24pt &quot;MS Sans Serif&quot;;

</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_back</zorder>
  <zorder>pushButton_resume</zorder>
  <zorder>pushButton_exit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
